main.js

```diff
@@ -31,7 +42,19 @@ const jitsiMeetWindowOptions = {
     height: 600,
     minWidth: 800,
     minHeight: 600,
-    titleBarStyle: 'hidden'
+    titleBarStyle: 'hidden',
+    webPreferences: { experimentalFeatures: true }
+};
+
+/**
+ * Options used when creating the micro mode window.
+ */
+const microWindowOptions = {
+    width: 800,
+    height: 600,
+    titleBarStyle: 'hidden',
+    frame: true,
+    webPreferences: { experimentalFeatures: true }
```

Comment:
we don't need this flag for the micro window itself, do we? The capture is done in the main window.